update_ssh_config.sh

```diff
@@ -1,30 +1,40 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -e
 
-function print_green(){
-    echo -e "\033[32m$1\033[39m"
+function print_green() {
+	echo -e "\033[32m$1\033[39m"
 }
 
-case "$(uname -s)" in
-
-   Darwin)
-     OS='Darwin'
-     ;;
-
-   Linux)
-     OS='Linux'
-     ;;
-
-   *)
-    echo "OS not supported"
-    exit
-    ;;
-esac
-
 REPO='git@bitbucket.org/magnet-cl/keygen.git'
 SSH_CONFIG_FILE='ssh_config'
 
+ADD_MACOS_CONFIG_FLAG=''
+IDENTITY_FILE_PATH=''
+
+print_usage() {
+	printf "Usage:
+\t-m
+\t\tAdds the following configuration to ssh hosts:\n
+\t\t\tAddKeysToAgent yes
+\t\t\tUseKeychain yes
+\t\t\tIdentitiesOnly yes\n
+\t\tTo ensure macos uses keychain.\n
+\t-i FILE
+\t\tSpecify identity file for ssh hosts.\n
+"
+}
+while getopts 'mi:' flag; do
```

Comment:
```bash
if [ $(uname) == "Darwin" ]; then
    ADD_MACOS_CONFIG_FLAG=1
fi
```
Es mucho más entendible.

Reply:
No es estrictamente necesaria la conf para que funcione en darwin, me acuerdo que la agregue hace harto pero no me acuerdo que fixeaba en realidad jajaja, voy a ver si me acuerdo pq la agregue

Reply:
github dice que tienen que estar esas líneas (https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key-for-a-hardware-security-key), entonces siguiendo el just werks aura, lo voy a dejar como dices 👍